File: fizzbuzz/jasmine/src/fizzBuzz.js
function fizzBuzz(x, y) {
  var array = []
  while (x <= y) {
    if (x % 3 == 0 && x % 5 == 0) {
      array.push("fizzbuzz");
    } else if (x % 3 == 0) {
      array.push("fizz");
    } else if (x % 5 == 0) {
        array.push("buzz");
    } else {
      array.push(x);
    }
    x += 1;
  }
  console.log(array)
}

File: fizzbuzz_dir/fizzbuzz.rb
def fizzbuzz(number)
  answer = ""
  if number % 3 == 0
    answer << "fizz"
  end
  if number % 5 == 0
    answer << 'buzz'
  end
  if number % 3 != 0 && number % 5 != 0
    return number
  end
  answer
end
